fmriprep/cli/parser.py

```diff
@@ -40,10 +40,19 @@ def _drop_sub(value):
         value = str(value)
         return value.lstrip('sub-')
 
+    def _filter_pybids_none_any(dct):
+        import bids
+        for k, v in dct.items():
+            if v is None:
+                dct[k] = bids.layout.Query.NONE
+            elif v == '__any__':
```

Comment:
What do you think about using a wildcard?

```suggestion
            elif v == '*':
```